cns/kubecontroller/nodenetworkconfig/conversion.go

```diff
@@ -92,7 +94,7 @@ func CreateNCRequestFromStaticNC(nc v1alpha.NetworkContainer) (*cns.CreateNetwor
 	for addr := primaryPrefix.Masked().Addr(); primaryPrefix.Contains(addr); addr = addr.Next() {
 		secondaryIPConfigs[addr.String()] = cns.SecondaryIPConfig{
 			IPAddress: addr.String(),
-			NCVersion: int(nc.Version),
+			NCVersion: 0,
```

Comment:
There is also a Version we stick in NC goal state and not on the IPConfig.

Reply:
Also this can impact VNETPrefix that will also use same mode but we need version check there

Reply:
return &cns.CreateNetworkContainerRequest{
		SecondaryIPConfigs:   secondaryIPConfigs,
		NetworkContainerid:   nc.ID,
		NetworkContainerType: cns.Docker,
		**Version:              strconv.FormatInt(nc.Version, 10), //nolint:gomnd // it's decimal**

Reply:
updated, setting the nc.Version = 0 and leaving everything else as is so that I don't have to track down every usage of it